content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -52,42 +59,50 @@ minio:
 
 It's also advisable to install the `s3-exporter` Helm chart, as it provides basic metrics about user cluster backups.
 
-#### Install Charts
+### Install Charts
 
 With this you can install the chart:
 
 ```bash
-helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace minio minio charts/minio/
-helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace s3-exporter s3-exporter charts/s3-exporter/
+helm --namespace minio upgrade --install --values /path/to/your/helm/values.yaml minio charts/minio/
+helm --namespace s3-exporter upgrade --install --values /path/to/your/helm/values.yaml s3-exporter charts/s3-exporter/
 ```
 
+## Add the Seed Resource
 
-### Add the Seed Resource
-
-To connect the new seed cluster with the master, you need to create a kubeconfig Secret and a Seed resource.
+To connect the new seed cluster with the master, you need to create a kubeconfig Secret and a Seed resource. This allows
+the KKP components in the master cluster to communicate with the seed cluster and reconcile user-cluster control planes.
 
-You will add the **master cluster** as the **seed cluster**
+{{% notice warning %}}
+To make sure that the kubeconfig stays valid forever, it must not contain temporary login tokens. Depending on the
+cloud provider, the default kubeconfig that is provided may not contain username+password / a client certificate, but instead
+try to talk to local token helper programs like `aws-iam-authenticator` for AWS or `gcloud` for the Google Cloud (GKE).
+These kubeconfig files **will not work** for setting up Seeds.
+{{% /notice %}}
 
-Make sure the kubeconfig contains static, long-lived credentials. Some cloud providers use custom authentication providers
-(like GKE using `gcloud` and EKS using `aws-iam-authenticator`). Those will not work in KKP’s usecase because the
-required tools are not installed inside the cluster environment.
+The Kubermatic repository provides a [script](https://github.com/kubermatic/kubermatic-installer/blob/release/v2.14/kubeconfig-serviceaccounts.sh) that can be used to prepare a kubeconfig for usage in Kubermatic. The script will create
```

Comment:
No, because the installer repository is EOL and will not be updated for 2.15 anymore (managing access to it was a PITA and we're lucky that now everything is public in the regular Kubermatic repo). Also, the script shoul at some point be migrated to the new installer (https://github.com/kubermatic/kubermatic/issues/5871), so linking to the 2.14 bash script is currently the best we can do here.